<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPost</class>
 <widget class="QMainWindow" name="VentanaPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana Post Ronda</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 245, 92)</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(83, 255, 246)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>541</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 60pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>RESUMEN DE RONDA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>50</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>sprites/Objetos/Cerveza.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Puntaje De Ronda:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Vida:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Cantidad Items Buenos:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Cantidad Items Peligrosos: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_puntaje_ronda">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>190</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_vida">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_items_buenos">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_items_malos">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>340</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="boton_continuar_jugando">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>470</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(215, 220, 220)</string>
      </property>
      <property name="text">
       <string>Continuar jugando</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boton_salir">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>470</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(215, 220, 220)</string>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cartel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>380</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(37, 255, 11);
font: 75 italic 14pt &quot;Optima&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="text">
       <string>             Puedes continuar jugando!!</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
